frb/galaxies/galfit.py

```diff
@@ -0,0 +1,480 @@
+
+        # Centroid
+        if position is None:
+            pos_x, pos_y = int(img.shape[0]/2), int(img.shape[1]/2)  
+        else:
+            pos_x, pos_y = position
+        fstream.write("1) {:d} {:d} 1 1 # position x y\n".format(pos_x,pos_y))
+
+        # What is the integrated magnitude?
+        if int_mag is None:
+            warnings.warn("No guess given for integrated magnitude. Proceeding with sum within region.")
+            cropped_img = img[region[2]:region[3],region[0]:region[1]]
+            # Better guess if other sources are masked out in badpix
+            if badpix !='none':
+                bad_pix = fits.getdata(badpix)
+                # Convert to boolean
+                bad_pix = np.where(bad_pix==0, True, False)
+                # Crop to region
+                bad_pix = bad_pix[region[2]:region[3],region[0]:region[1]]
+            else:
+                bad_pix = True
+            int_mag = zeropoint-2.5*np.log10(np.sum(cropped_img[bad_pix]))
+        fstream.write("3) {:f} 1 # Integrated magnitude\n".format(int_mag))
+        # Effective radius
+        if r_e is None:
+            warnings.warn("Guess for r_e not given. This might not converge.")
+            r_e = (region[3]-region[2]+region[1]-region[0])/2/3 #A third of the average region dimension
+        fstream.write("4) {:f} 1 # effective radius (pix)\n".format(r_e))
+        # Sersic index
+        fstream.write("5) {:f} 1 # sersic index\n".format(n))
+        # Blanks
+        for num in [6,7,8]:
+            fstream.write("{:d}) 0.0000 0 # ----\n".format(num))
+        # b/a
+        fstream.write("9) {:f} 1 # Axis ratio (b/a)\n".format(axis_ratio))
+        # PA of the galaxy
+        fstream.write("10) {:f} 1 # Position angle (PA) [deg: Up=0, left =90]\n".format(pa))
+        # Don't skip sersic fitting at any cost.
+        fstream.write("Z) 0 # Skip this model? (yes=1,no=0)\n\n")
+        # Skip sky fitting though?
+        if not skip_sky:
+            # Just a flat sky. No fancy gradients.
+            fstream.write("# Component number: 2\n")
+            fstream.write("0) sky # component type\n")
+            _, median, _ = sigma_clipped_stats(img)
+            fstream.write("1) {:f} 1 # Sky background\n".format(median))
+            fstream.write("2) 0 0 # dsky/dx\n")
+            fstream.write("3) 0 0 # dsky/dy\n")
+            fstream.write("Z) 0 # Skip this model\n")
+        fstream.write("================================================================================\n")
+    # Done.
+    return configfile
+
+def read_fitlog(outfile:str, initfile:str)->dict:
+    """
+    Reads the output fit.log
+    file and returns the sersic fit parameters
+    Args:
+        outfile (str): Path and name of the fit log
+            file.
+        initfile (str): Path to the config file
+            used to produce the log entry. This
+            is used to distinguish multiple
+            entried in the logfile.
```

Comment:
Add a brief description of what is in the returned `dict`

Reply:
Done.